pkg/constants/constants.go

```diff
@@ -79,6 +79,9 @@ const (
 	// this annotation will be cleared
 	WorkspaceStopReasonAnnotation = "controller.devfile.io/stopped-by"
 
+	// WebhookRestartedAtAnnotation holds the the time (unixnano) of when the webhook pod was restarted
+	WebhookRestartedAtAnnotation = "controller.devfile.io/restart-at"
```

Comment:
as discussed in another PR
```suggestion
	WebhookRestartedAtAnnotation = "controller.devfile.io/restarted-at"
```

Reply:
fixed!